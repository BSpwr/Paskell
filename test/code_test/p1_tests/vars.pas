program VarsTest;
var
   grade: string = 'A';
   good: boolean = false;
   junk, junk2: string;
   val: real = 5.0;
   b: real = 5.0;
   s, a: real;
begin

    grade := 'Z';
    junk := 'A';
    junk2 := 'B';
    s := 5;
    a := 10;
    b := b * b * b;

    writeln(grade, ' ', good);
    writeln(junk, ' ', junk2);
    writeln(s);
    writeln(a);
    writeln(b);

end.