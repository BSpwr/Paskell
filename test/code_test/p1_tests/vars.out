Z false
A B
5.0
10.0
125.0